src/config.rs

```diff
@@ -316,14 +314,22 @@ impl<'de> Deserialize<'de> for Decorations {
                 match value.to_lowercase().as_str() {
                     "none" => Ok(Decorations::None),
                     "full" => Ok(Decorations::Full),
-                    "transparent" => {
-                        eprintln!("macos-only decorations value: {}; Using default value", value);
+                    "true" => {
+                        eprintln!("deprecated decorations boolean value, use one of \
+                                   default|transparent|buttonless|none instead; \
+                                   Falling back to \"full\"");
                         Ok(Decorations::Full)
                     },
-                    "buttonless" => {
+                    "false" => {
+                        eprintln!("deprecated decorations boolean value, use one of \
+                                   default|transparent|buttonless|none instead; \
+                                   Falling back to \"none\"");
+                        Ok(Decorations::None)
+                    },
+                    "transparent" | "buttonless" => {
                         eprintln!("macos-only decorations value: {}; Using default value", value);
```

Comment:
Does it really make sense to include `transparent` and `buttonless` in the suggested values, when using either of those also results in a warning and a fallback?

Reply:
It does not, thanks for the heads-up. I was just copy-pasting things from macOS without giving it too much thought. The recommendations for linux should only be `full` and `none`.

However, now that you mention this, the `default` recommendation should be removed from both platforms, since that is not a valid value for either.